portals/publisher/source/src/app/components/Apis/Details/Properties/Properties.jsx

```diff
@@ -769,6 +769,7 @@ function Properties(props) {
                                                 </Button>
                                             ) : (
                                                 <CustomSplitButton
+                                                    api={api}
```

Comment:
use a null check for advertiseInfo

Reply:
Done with https://github.com/wso2/apim-apps/pull/130/commits/bab2439306d03e8f1775c16d276683ba43e558dd